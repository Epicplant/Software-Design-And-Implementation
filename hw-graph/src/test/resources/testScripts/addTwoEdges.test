# Tests adding two edges pointing back and forth between between 2 nodes

# Creates a new graph
CreateGraph newGraph

# Creates the edges' first node
AddNode newGraph n1

# Creates the edges' second node
AddNode newGraph n2

# Creates an edge from n1 to n2 with weight 2
AddEdge newGraph n1 n2 2

# Creates an edge from n2 to n1 with weight -2
AddEdge newGraph n2 n1 -2

# Lists all of the children of this node
ListChildren newGraph n1

# Lists all of the children of this node
ListChildren newGraph n2