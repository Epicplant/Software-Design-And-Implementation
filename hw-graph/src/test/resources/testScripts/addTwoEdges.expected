# Tests adding two edges pointing back and forth between between 2 nodes

# Creates a new graph
created graph newGraph

# Creates the edges' first node
added node n1 to newGraph

# Creates the edges' second node
added node n2 to newGraph

# Creates an edge from n1 to n2 with weight 2
added edge 2 from n1 to n2 in newGraph

# Creates an edge from n2 to n1 with weight -2
added edge -2 from n2 to n1 in newGraph

# Lists all of the children of this node
the children of n1 in newGraph are: n2(2)

# Lists all of the children of this node
the children of n2 in newGraph are: n1(-2)